lottie-ios.podspec

```diff
@@ -24,6 +24,7 @@ For the first time, designers can create and ship beautiful animations without a
 
   s.ios.deployment_target = '8.0'
   s.osx.deployment_target = '10.10'
+  s.tvos.deployment_target = '9.0'
 
   s.source_files = 'lottie-ios/Classes/**/*'
```

Comment:
Does this work on tvOS without crashing? I haven't been up to date on the supported features of tvOS @romainbiard